docs/supported_workflows.md

```diff
@@ -152,9 +153,10 @@ The CLI currently supports the following operations on these `pulp_rpm` objects
 (C = Create, R = Read, U = Update, D = Delete):
 
 * RPM Alternative Content Sources - **CRUD, Add, Remove, Refresh**
-* RPM Packages Content - **CR**
+* RPM Package/Advisory/ModuleMD Defaults/ModuleMD Content - **CR**
+* RPM Distribution Tree/Repo Metadata Content - **R**
 * RPM Distributions - **CRUD**
 * RPM Publications - **CRD**
-* RPM Remote - **CRUD**
+* RPM/ULN Remote - **CRUD**
 * RPM Repository - **CRUD, Modify, Sync**
```

Comment:
So we are missing RBAC on RPM... I see a prio-list here.

Reply:
To be fair, RBAC only got released for RPM in rpm/3.19 on 2023-02-06